user_tools/src/spark_rapids_tools/cmdli/tools_cli.py

```diff
@@ -287,6 +288,33 @@ def train(self,
                          wrapper_options=train_args)
         tool_obj.launch()
 
+    def stats(self,
+              config_path: str = None,
+              output_folder: str = None,
+              qual_output: str = None):
+        """The stats cmd generates statistics from the qualification tool output.
+        :param config_path: Path to the configuration file.
+        :param output_folder: Path to store the output.
+        :param qual_output: path to the directory, which contains the qualification tool output.
+                            E.g. user should specify the parent directory $WORK_DIR where
+                            $WORK_DIR/rapids_4_spark_qualification_output exists.
```

Comment:
>  path to the directory, which contains the qualification tool output.
                            E.g. user should specify the parent directory $WORK_DIR where
                            $WORK_DIR/rapids_4_spark_qualification_output


Does this mean that we pass `qual_2024xxx`?

The tool does not seem to detect the correct files if the parent directory `qual_2024xxx` is passed: 
```
spark_rapids stats -q /tools-run/qual_20240729205415_A6e8D9E7
```

Output:
```
ERROR rapids.tools.qualification.stats: Error reading CSV files: [Errno 2] No such file or directory: '/tools-run/qual_20240729205415_A6e8D9E7/rapids_4_spark_qualification_output_unsupportedOperators.csv'
```